src/Admin/Fields.php

```diff
@@ -151,18 +144,16 @@ public function save_fields_product_option( $wc_product ) {
 		foreach ( $this->fields as $field ) {
 			$field_post_data = $_POST[ $field['id'] ]; //phpcs:ignore
```

Comment:
Il faudrait vérifier si l'id est `isset` dans $_POST désormais comme on a retiré le check via le empty.

Reply:
Noté, je traiterai avec le commentaire au-dessus